cypress/integration/finance/funds/funds.deleteBudget.spec.js

```diff
@@ -0,0 +1,89 @@
+import uuid from 'uuid';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import NewFund from '../../../support/fragments/finance/funds/newFund';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import { getCurrentFiscalYearCode } from '../../../support/utils/dateTools';
+import { testType } from '../../../support/utils/tagTools';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import TopMenu from '../../../support/fragments/topMenu';
+
+describe('ui-finance: Delete budget from fund', () => {
+  const currentBudgetSectionId = 'currentBudget';
+  const ledger = {
+    id: uuid(),
+    name: `autotest_ledger_${getRandomPostfix()}`,
+    code: `autotest_code_${getRandomPostfix()}`,
+    description: `autotest_ledger_ description_${getRandomPostfix()}`,
+    ledgerStatus: 'Frozen',
+    currency: 'USD',
+    restrictEncumbrance: false,
+    restrictExpenditures: false,
+    acqUnitIds: '',
+    fiscalYearOneId: ''
+  };
+
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+
+    cy
+      .okapiRequest({
+        path: 'acquisitions-units/units',
+        limit: 1,
+      })
+      .then(({ body }) => {
+        ledger.acqUnitIds = [body.acquisitionsUnits[0].id];
+      });
+
+    cy
```

Comment:
may be It's a good time to create several additional cypress custom actions to new requests processing